babel.config.js

```diff
@@ -4,28 +4,31 @@ module.exports = {
   babelrcRoots: ['examples/*'],
   overrides: [
     {
-      presets: ['@babel/preset-flow'],
+      presets: [require.resolve('@babel/preset-flow')],
```

Comment:
Otherwise Babel would resolve these from e2e dirs that are not Yarn workspaces, where they are not installed as deps